lib/ansible/plugins/connection/ssh.py

```diff
@@ -316,24 +316,6 @@
         vars:
             - name: ansible_ssh_transfer_method
               version_added: '2.12'
-      scp_if_ssh:
```

Comment:
Not sure how well this test works since the removed option doesn't cause an error, but https://github.com/ansible/ansible/blob/devel/test/integration/targets/connection_ssh/runme.sh#L69 should use `ANSIBLE_SSH_TRANSFER_METHOD=scp` now.

Reply:
(or, ANSIBLE_SCP_IF_SSH=true is outdated at least)